services/src/main/scala/cromwell/pipeline/service/AuthService.scala

```diff
@@ -48,6 +46,15 @@ object AuthService {
 
       import authUtils._
 
+      private def takeUserFromRequest(request: SignInRequest): OptionT[Future, UserWithCredentials] =
```

Comment:
Usually we place private methods at bottom

Reply:
Thanks, fixed